현정/5주차_11047.java

```diff
@@ -0,0 +1,39 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.StringTokenizer;
+
+public class BJ_11047 {
+    static int n;
+    static int k;
+    static int[] coin;
+
+    static int greedy() {
+        int cnt = 0;
+        int remain = k;
+        for (int i = n - 1; i >= 0; i--) {
+            //큰 동전부터 나눠서 동전 개수 더하기
+            cnt = cnt + remain / coin[i];
+            //나머지 값 갱신
+            remain = remain % coin[i];
```

Comment:
```suggestion
            cnt += remain / coin[i];
            //나머지 값 갱신
            remain %= coin[i];
```
자바도 이거 될거같은데 이런식으로 줄이는게 좀 더 좋아보이네용